modules/client/examples/06-multisig-client/09-get-proposal.ts

```diff
@@ -16,15 +16,16 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 // Create an multisig client
 const client = new MultisigClient(contextPlugin);
 
-const proposalId = "0x12345...";
+const proposalId =
+  "0x1234567890123456789012345678901234567890_0x0000000000000000000000000000000000000000000000000000000000000000";
 
 const proposal: MultisigProposal | null = await client.methods.getProposal(
   proposalId,
 );
 console.log(proposal);
 /*
 {
-  id: "0x12345...",
+  id: "0x1234567890123456789012345678901234567890_0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Let's follow the Subgraph format if possible, and save nonce chars if not strictly necessary. The only requirement is that nonce values can be parsed back non-ambiguously (related to potential odd-length hex values)

Reply:
changed